src/locale/en.json

```diff
@@ -0,0 +1,55 @@
+{
+  "page-title": "COVID-19 Housing Protection Legislation and Housing Justice Action Map",
+  "about-title": "COVID-19 Housing Protection Legislation & Housing Justice Action",
+  "about-map": "About the map:",
+  "about-protections": "Housing Protection Legislation",
+  "about-description": "Where officials have passed (or tenants are working to pass) housing protection legislation during the COVID-19 crisis.",
+  "legislation-form-usa": "Submit housing protection legislation for locations in the United States.",
+  "legislation-form-intl": "Submit housing protection legislation for locations outside of the United States.",
+  "policyStrength": {
+    "1": "Few protections in place",
+    "2": "Some protections in place",
+    "3": "Many protections in place"
+  },
+  "housing-action-title": "Housing Justice Action",
+  "housing-action-description": "Where <strong class='color-rent-strike'>housing justice actions</strong> are occurring or have occured.",
+  "housing-action-form": "Fill out to submit information about housing justice action (such as rent strikes, occupations, mutual aid).",
+  "resources-title": "Resources",
+  "resources-description": "Here are some resources to get help if you are in need.",
+  "resources": {
+    "right-to-the-city": "Resources and demands from Right To The City Alliance and Homes For All.",
+    "european-action-coalition": "Resources and demands from the European Action Coalition for the Right to Housing.",
+    "hashtag-cancel-rent": "Rent Strike resources for #CancelRent."
+  },
+  "infowindow": {
+    "policy": {
+      "title": "Housing Protection Legislation",
+      "strength-label": "Policy Strength:",
+      "summary-label": "Policy Summary:",
+      "start-date-label": "Effective: ",
+      "end-date-label": "Ends: ",
+      "link": "View more info.",
+      "jurisdictionType": {
+        "city": "City",
+        "county": "County",
+        "state-province": "State/Province",
+        "nation": "Nation"
+      }
+    },
+    "action": {
+      "title": "Housing Justice Action",
+      "loc-label": "Location: ",
+      "status-label": "Status: ",
+      "why-label": "Why? ",
+      "start-date-label": "Start: ",
+      "resources-label": "Resources: "
+    }
+  },
+  "layer-select": {
```

Comment:
FYI in a previous PR these were updated so that their meaning is more clear:

```
Housing Justice Actions
City Protections
County Protections
State/Province Protections
National Protections
```

we should update the translations accordingly (cc @unixjazz)

Reply:
Sounds good :)

Reply:
Done

Reply:
Thanks!